Source/PrivateHeaders/Ably/ARTRealtimeChannel+Private.h

```diff
@@ -37,6 +38,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (BOOL)shouldAttach;
 - (ARTChannelProperties *)properties_nosync;
 
+// The channel that this internal object belongs to. We need a reference to it so that we can pass it to plugins for them to handle LiveObjects protocol messages. Avoid using this property for anything else. TODO understand whether this needs to be weak in https://github.com/ably/ably-cocoa-liveobjects-plugin/issues/9
+@property (nonatomic, weak) ARTRealtimeChannel *channel;
```

Comment:
Can't it be `id<ARTRealtimeChannelProtocol> *channelRefForPlugins` so its usage would be obvious and to avoid keeping reference to `ARTRealtimeChannel` inside `ARTRealtimeChannelInternal` which looks a bit strange.